### PR TITLE
running on virtualbox

### PR DESCRIPTION
**Pull request by simrat12**
Brief explanation of how to run this repo using Virtualbox with an Ubuntu ISO file.

### CODE REVIEW

**Comment by samelamin** [comment]
Thanks for this @simrat12 , can you merge/rebase master into this to resolve conflicts?

File: README.md [hunk 1]
```diff
@@ -14,41 +14,16 @@ To go through this guide you will need:
...
-From the top-level of the repo.
+1. Install VirtualBox from https://www.virtualbox.org/wiki/Downloads
+2. Download Ubuntu ISO file from https://ubuntu.com/download 
+3. Boot the VM from the ISO file, a good guide is here: https://www.youtube.com/watch?v=J1Bf8FM8yN8 
+4. Install yarn on the VM: https://linuxize.com/post/how-to-install-yarn-on-ubuntu-20-04/ 
+5. CD into the API directory 
+6. Run make
+7. Run yarn start
 
-1. Add some env variables to your bash profile and source.
```
Comment (f-gate) [inline README.md:21]: Hold on... This is removing the unix/docker instructions?

Can we keep those please!! haha.

**Review by f-gate — CHANGES_REQUESTED** [review]
see comment

**Review by f-gate — APPROVED** [review]
thanks!

**Comment by samelamin** [comment]
thanks @simrat12

**Pull request merged**